src/rgw/rgw_kms.cc

```diff
@@ -0,0 +1,275 @@
+// -*- mode:C; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+/**
+ * Server-side encryption integrations with Key Management Systems (SSE-KMS)
+ */
+
+#include <rgw/rgw_crypt.h>
+#include <rgw/rgw_keystone.h>
+#include <rgw/rgw_b64.h>
```

Comment:
can #include as "rgw_*.h"

Reply:
Again copy-pasted from rgw_crypto but I'll fix it